lib/config-generator.js

```diff
@@ -67,8 +68,13 @@ class ConfigGenerator {
 
         config.stats = this.buildStatsConfig();
 
+        let extensions = ['.js', '.jsx'];
+        if (this.webpackConfig.useTypeScriptLoader) {
+            extensions = extensions.concat(['.json', '.ts', '.tsx']);
+        }
```

Comment:
I see, that's very good to have. Could you please point me where this validation occur? I just can not find it now :(